apps/public/.env

```diff
@@ -0,0 +1,3 @@
+PUBLIC_URL="http://localhost:3000"
+CLIENT_URL="http://localhost:3001"
+ADMIN_URL="http://localhost:3002"
```

Comment:
If we end up putting sensitive data in here, we'll need a more realistic .env workflow, but I just wanted to get this up and running for now.